packages/boot/src/__tests__/acceptance/component-application.booter.acceptance.ts

```diff
@@ -97,6 +124,14 @@ describe('component application booter acceptance tests', () => {
     }
   }
 
+  class MainAppWithSubAppBooter extends BootMixin(Application) {
+    constructor() {
+      super();
+      this.projectRoot = __dirname;
+      this.applicationBooter(app);
```

Comment:
Should the method name start with a verb, e.g. `this.addApplicationBooter`?

Reply:
Again, I choose a noun to be consistent with other methods.